app/models/course.rb

```diff
@@ -10,8 +10,9 @@
 #
 
 class Course < ApplicationRecord
-  has_many :students
+	has_many :courses_students
+  has_many :students, :through => :courses_students
   has_and_belongs_to_many :teachers
-  has_many :attendances
+  #has_many :attendances
```

Comment:
Remember to not leave comments in code being merged into master. Merging into master should mean production ready.

Reply:
Fixed